src/driver/koa/KoaDriver.ts

```diff
@@ -43,7 +43,7 @@ export class KoaDriver extends BaseDriver {
         const bodyParser = require("koa-bodyparser");
         this.koa.use(bodyParser());
         if (this.cors) {
-            const cors = require("kcors");
+            let cors = require("@koa/cors") || require("kcors");
```

Comment:
What is the behavior of `require` when module doesn't exist? It throws error or return falsy value?